locales/en/allocations.json

```diff
@@ -24,5 +24,40 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{moves_count}} {{person_or_people}} from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "allocation_tab": "Allocation",
+    "allocation_history_tab": "Allocation history",
+    "fields": {
+      "estate_type": "Estate type",
+      "category": "Category",
+      "complex_cases": "Complex cases for prisons to agree",
+      "none_provided": "None provided",
+      "other_criteria": "Other criteria",
+      "sentence_length": {
+        "label": "Time left to serve",
+        "value": "Any length of sentence",
+        "value_short": "16 months or less to serve",
+        "value_long": "16 months or longer to serve"
+      }
+    },
+    "complete_in_full": "This allocation should be filled in full - consider prisoners that are the next best for this allocation",
+    "people": {
+      "slots_filled": "{{slots_filled}} of {{total_slots}} {{person_or_people}} allocated",
+      "heading": "People for allocation",
+      "empty_slot": "Prisoner to be allocated"
```

Comment:
We're using a combination of `People` and `Prisoner` between `heading` and `empty_slot`. We should probably stick to one term.

Reply:
I've gone backwards and forwards on this..

Allocations are ONLY for prisoners (not detainees). 

Problem with people is it is a bit ambiguous what is meant is meant by people - ie. is it prisoners.... But it's more humanising than the term prisoners.

Problem with prisoners is it's too long and causes problems in things like titles. But it provides far more clarity.

I've deliberately added the term prisoner in detail, and kept the term people in the title to try and get the best of both worlds. ie. Clearer when we are asking specific questions (prisoner), shorter and more humanising in titles (people). 

If we go for one, it should be people, but I'm concerned it might cause a bit of confusion. We do refer to prisoners as people elsewhere in other journeys though.

Reply:
Ok -- so what is my action here? Is change needed?

Reply:
Just go for people everywhere. But if there is problems, may need to change later

Reply:
👍